src/main/app.ts

```diff
@@ -45,6 +45,8 @@ import { UpdateChecker } from './update-checker';
 declare const MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY: string;
 declare const MAIN_WINDOW_WEBPACK_ENTRY: string;
 
+let turnShortcutsVar = true;
```

Comment:
Please do not use global variables. Use instance properties instead. For example around line 78. And please add a comment what the property is used for.